pkg/engine/vmextensions.go

```diff
@@ -51,13 +51,17 @@ func CreateCustomScriptExtension(cs *api.ContainerService) VirtualMachineExtensi
 	name := "[concat(variables('masterVMNamePrefix'), copyIndex(variables('masterOffset')),'/cse', '-master-', copyIndex(variables('masterOffset')))]"
 	outBoundCmd := ""
 	registry := ""
+	ncBinary := "nc"
+	if cs.Properties.MasterProfile != nil && cs.Properties.MasterProfile.IsCoreOS() {
```

Comment:
Meh, the func name `CreateCustomScriptExtension` is misleading, as it suggests it's generic. Reading through the code it seems that this is only invoked to compose CSE for master vms. Just a note to self we should rename this to `CreateMasterVMCustomScriptExtension` or something.